tests/Exercise/BettysBikeShopTest.elm

```diff
@@ -15,7 +15,7 @@ tests =
     describe "BettysBikeShopTest"
         [ exemplar
         , otherSolutions
-        , noFuctionSignature
+        , noFunctionSignature
```

Comment:
It looks like these are whitespace changes, but probably better to remove them from the PR

Reply:
It's a fly-by typo fix. "Fuction" was missing its N. I'll leave it in.